hippunfold/run.py

```diff
@@ -1,24 +1,27 @@
 #!/usr/bin/env python3
-import os
+from pathlib import Path
 
-from snakebids.app import SnakeBidsApp
-from snakebids.cli import add_dynamic_args
+from snakebids import bidsapp, plugins
 
+if '__file__' not in globals():
```

Comment:
not sure why this is needed? it worked fine without for me?

Reply:
I had to add this to pass the readthedocs check; it has more to do with the documentation than the application itself.

Reply:
ah ok then we can leave it as you have it..